apps/src/p5lab/AnimationTab/PiskelEditor.jsx

```diff
@@ -35,7 +35,8 @@ class PiskelEditor extends React.Component {
     onNewFrameClick: PropTypes.func.isRequired,
     pendingFrames: PropTypes.object,
     removePendingFrames: PropTypes.func.isRequired,
-    isBlockly: PropTypes.bool
+    isBlockly: PropTypes.bool,
+    locales: PropTypes.object.isRequired
```

Comment:
```suggestion
    locales: PropTypes.string
```

Reply:
I put an object since in the picture attached here it shows an object is stored in the redux store. 

I thought it had to be required since the piskel editor will need the locale code to get the correct translation loaded. But if not I can remove the isRequired flag

![image](https://user-images.githubusercontent.com/18276475/117053944-6a348780-acce-11eb-9856-ec56ddc5e4ac.png)